### PR TITLE
WebGLRenderer helpers renderPass, renderOverride (attempt #3)

### PR DESCRIPTION
**Pull request by bhouston**
This PR adds these two helper functions.  Reduces the need for boilerplate to get basic passes working.

This time I did not do anything with the clearColor, clearAlpha that prevented the last version from being merged.

### CODE REVIEW

File: src/renderers/WebGLRenderer.js [hunk 1]
```diff
@@ -1166,6 +1168,30 @@ THREE.WebGLRenderer = function ( parameters ) {
...
+		if( passScene === null ) {
+			passCamera = new THREE.OrthographicCamera( -1, 1, 1, -1, 0, 1 );
+			passQuad = new THREE.Mesh( new THREE.PlaneBufferGeometry( 2, 2 ), null );
+			passScene = new THREE.Scene();
+			passScene.add( passQuad );
+		}
+
+		passQuad.material = passMaterial;
+		this.render( passScene, passCamera, renderTarget, forceClear );
+		passQuad.material = null;
```
Comment (WestLangley) [inline src/renderers/WebGLRenderer.js]: Hmm... The last three lines are the same as

    this.renderOverride( passMaterial, passScene, passCamera, renderTarget, forceClear );

Comment (bhouston) [inline src/renderers/WebGLRenderer.js]: I've addressed this change!  Thanks for pointing out the code savings. :)

**Comment by mrdoob** [comment]
I still think it would be better to call this `renderMaterial`...
I'm adding `scene.background` and we could probably wire stuff up.

**Comment by mrdoob** [comment]
2baec79aef70921c439a3e9ca15b016494a5fd73

**Comment by bhouston** [comment]
I've updated this PR with current dev and renamed renderOverride to renderMaterial -- that was a good suggestion.

**Comment by bhouston** [comment]
oops.  sec. I'm on the wrong branch.

**Comment by bhouston** [comment]
There we go.  This PR is updated with your requested changes and mergeable to dev branch.

**Pull request closed without merging**